lib/Common/ConfigFlagsList.h

```diff
@@ -1477,6 +1487,9 @@ FLAGR(Number, JITServerIdleTimeout, "Idle timeout in seconds to do the cleanup i
 FLAGR(Number, JITServerMaxInactivePageAllocatorCount, "Max inactive page allocators to keep before schedule a cleanup", 10)
 
 FLAGNR(Boolean, StrictWriteBarrierCheck, "Check write barrier setting on none write barrier pages", false)
+FLAGNR(Boolean, WriteBarrierTest, "Always return true while checking barrier to test recycler regardless of annotation", DEFAULT_CONFIG_WriteBarrierTest)
+FLAGNR(Boolean, EnableHardwareWriteBarrier, "Use to turn off write watch to test software write barrier on windows", DEFAULT_CONFIG_EnableHardwareWriteBarrier)
```

Comment:
looks my codeflow can't post message to the github

"my original idea was this, with GLOBAL_FORCE_USE_WRITE_BARRIER is on on windows,  we can change with this flag to test with either software write barrier or hardware write barrier. if we found a issue in software write barrier, we can use the same binary to verify if this issue can repro with hardware write barrier. do you think this is useful? if yes I suggest we keep this until the feature is done."